documentation/en/default-lotus-config.toml

```diff
@@ -226,3 +226,77 @@
     #ColdStoreRetention = 0
 
 
+[Raft]
```

Comment:
Maybe call this group "Cluster"? Raft is something most people will need to look up to know what it's doing.